gensim/models/fasttext.py

```diff
@@ -1264,8 +1264,21 @@ def _load_fasttext_format(model_file, encoding='utf-8', full_model=True):
     model.vocabulary.raw_vocab = m.raw_vocab
     model.vocabulary.nwords = m.nwords
     model.vocabulary.vocab_size = m.vocab_size
-    model.vocabulary.prepare_vocab(model.hs, model.negative, model.wv,
-                                   update=True, min_count=model.min_count)
+
+    #
+    # We explicitly set min_count=1 regardless of the model's parameters to
```

Comment:
Feel free to include a link to the original Github issue (useful for additional context in the future).

Reply:
OK, done